google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -589,6 +589,15 @@ public Builder setSystemExecutorProvider(ExecutorProvider executorProvider) {
       return this;
     }
 
+    /**
+     * Sets the number of pullers used to pull messages from the subscription. Defaults to the
+     * number of available processors.
+     */
+    public Builder setNumPullers(int numPullers) {
```

Comment:
Do you have preference between `setParallelPullCount` and `setNumStreams`? I like the latter for being a little shorter but that's not a very important metric.